Add intial service info docs to README (#1)
cloud-gov/aws-elasticsearch-example: python/README.md

Python Example
==============

> The following with example CF deployment is an overview of the new AWS Elasticsearch service and highlights the differences between the legacy `elasticsearch56` and new `aws-elasticsearch` services available in the cloud.gov cloudfoundry marketplace.

Reviewer: Small grammar adjustment:

```suggestion
The following example CF deployment is an overview of the new AWS Elasticsearch service and highlights the differences between the legacy `elasticsearch56` and new `aws-elasticsearch` services available in the cloud.gov marketplace.
```

## Getting started

> The following example uses two services already created and available for consumption to your organization and space applications. The following services created for this example are named `old-es-service` (legacy elasticsearch56 service) and `new-es-service` (new aws-elasticsearch service).  Before running through this example, make sure to create the two services with the following cli commands:

Reviewer: This suggestion might make this read a little smoother:

```suggestion
This example uses two services already created and available for consumption to your organization and space applications. The services created for this example are named `old-es-service` (legacy elasticsearch56 service) and `new-es-service` (new aws-elasticsearch service).

First, create the two services with the following CLI commands:
```

```bash
##
## Creating the following example services
##

### The legacy elasticsearch service
$ cf create-service elasticsearch56 medium old-es-service

### The new elasticsearch service
$ cf create-service aws-elasticsearch BETA-es-dev new-es-service
```

> To get started with this example, please clone or download this repository so you can follow along and run the steps from your computer.

Reviewer: Same suggestion as above, we're continuing our steps here:

```suggestion
To continue, please clone or download this repository so you can follow along and run the steps from your computer:
```

```bash
## Clone and navitgate into example repository
$ git clone https://github.com/cloud-gov/aws-elasticsearch-example && cd aws-elasticsearch-example
```

## Deploying

> The following deployment instructions assumes that you have cloned or downloaded this `aws-elasticsearch-example` and have navigated into the root of this directory (`$ cd python/`).  From the `python` directory push the CF deployment to you cloud.gov space.
> 
> ```bash
> $ cf push -f manifest.yml

Reviewer: Same suggestion as above to help streamline this:

```suggestion
From the root repository directory (`aws-elasticsearch-example`), navigate into the `python` directory and push the CF deployment to you cloud.gov space:

```bash
# Navigate to the Python subdirectory
$ cd python

# Push the application
$ cf push -f manifest.yml
```

```

> The following apps are deployed into your space:
> - `elastic-tasks` is a CF app task runner used to load data and indices into the elasticsearch services.

Reviewer: Same suggestions as above regarding this list:

```suggestion
When the `cf push` command finishes, the following apps are deployed into your cloud.gov space:
- `elastic-tasks` - an app task runner that is used to load data and indices into the Elasticsearch services.
```

Or, if this is the only app, perhaps this:

```suggestion
When the `cf push` command finishes an `elastic-tasks` app task runner will be deployed, which is used to load  data and indices into the Elasticsearch services.
```

## Running the task

> To test loading some sample sample data, we will run the task. This will run a simple python script found in the [`run`](./run) file grabs our credentials to the new AWS Elasticsearch service, uploads a document and the gets that document back from the service instance.

Reviewer: This might be clearer to read with these suggestions:

```suggestion
With the `elastic-tasks` app deployed we can run the task to test loading sample data.  This involves running a small Python script found in the [`run`](./run) directory, which performs the following steps:

- Retrieves our credentials to the new AWS Elasticsearch service
- Uploads a document
- Retrieves the document back from the service instance

We can start the task with the following command:
```

Hopefully I got the steps right! :-)

Also, not that it technically matters given the shebang line and how things are called, but shouldn't the file name be `run.py`?

```bash

> ## Run the task and all the run script

Reviewer: Spelling nit:

```suggestion
## Run the task and call the run script
```
